build.gradle.kts

```diff
@@ -1,17 +1,38 @@
+import org.jetbrains.intellij.platform.gradle.TestFrameworkType
+import org.jetbrains.intellij.platform.gradle.tasks.VerifyPluginTask
+
 plugins {
-    id("org.jetbrains.intellij") version "1.17.4"
+    id("org.jetbrains.intellij.platform") version "2.0.1"
     java
     kotlin("jvm") version "2.0.20"
 }
 
 group = "com.asyncapi.plugin.idea"
-version = "2.5.0+jre17"
+version = "2.6.0+jre17"
 
 repositories {
     mavenCentral()
+    intellijPlatform {
+        jetbrainsRuntime()
+        defaultRepositories()
+    }
 }
 
 dependencies {
+    intellijPlatform {
+        intellijIdeaCommunity("2024.2.0.2", useInstaller = false)
```

Comment:
Will we need to specify a *new* IDE _Community_ version every time?

After all, see below: we specify every IDE version under the sun to allow for backwards compatibility.

Reply:
I'll rollback it to 2023.3 because of issues with byte-code compatibility

> Our developers believe that you likely built the plugin against version 2024.2, which includes a companion object in this class. As a result, the generated bytecode references it. To ensure compatibility, the plugin should be built against the lowest supported version, which in this case is 2022.3.
> 
>Please adjust the IntelliJ version to 2022.3 in the Gradle build script and try building the plugin again.